templates/reset-password.hbs

```diff
@@ -0,0 +1,16 @@
+{{# if msg}}
+<div class="msg">{{ msg }}</div>
+{{/if}}
+
```

Comment:
It would be nice if there was a better description on this page. For example:

```
<p>You forgot your password! Don't worry, it happens to the best of us.</p>
<p>To reset your password, enter your email address below. If we had a record of your email address, we will send you an email with a link that allows you to reset your password.</p>
```